Fix ASpace DIP upload

jraddaoui opened the pull request:
The ASpace config dict is used in linkTaskManagerReplacementDicFromChoice
to set the parameters sent to the upload-archivesspace script, which is
expecting restrictions instead of premis. This field doesn't seem to
be used anywhere else, so it only needs to be updated in the related form
fields that are used to update the config dict.

This also fixes an issue where the config port was not being used in the upload.

Fixes https://github.com/artefactual/archivematica/issues/746

sevein commented:
@jraddaoui are you still working on this?

jraddaoui commented:
I've asked for feedback in #746

jhsimpson reviewed (approved):
I tested this with @jraddaoui and it looks good.  :+1:

jraddaoui committed:
Fix ASpace DIP upload, refs #746

The ASpace config dict is used in linkTaskManagerReplacementDicFromChoice
to set the parameters sent to the upload-archivesspace script, which is
expecting restrictions instead of premis. This field doesn't seem to
be used anywhere else, so it only needs to be updated in the related form
fields that are used to update the config dict.

This also fixes an issue where the config port was not being used in the upload.

jraddaoui commented:
fd6947c437ac3bd35a481bc023fddeaee3c3043a in qa/1.x
